attachments/forms.py

```diff
@@ -66,3 +73,23 @@ def formfield(self, prop, field_class=None, **kwargs):
         defaults.update(kwargs)
         field = field_class(**defaults)
         return field
+
+    def validate_attachment(self, instance, property):
+        if not property.allowed_file_types:
+            return
+        # Checking if file extension is within allowed extension list
+        allowed_exts = property.allowed_file_types.split(' ')
```

Comment:
I'd probably just use `.split()` with no arguments and let it split on any whitespace.